packages/react-strict-dom/src/dom/runtime.js

```diff
@@ -123,47 +123,47 @@ const textarea: StrictReactDOMPropsStyle = [
 const ul: StrictReactDOMPropsStyle = styles.block;
 
 export const defaultStyles = {
-  a,
-  article,
-  aside,
-  b,
-  bdi,
-  bdo,
-  blockquote,
-  br,
-  button,
-  code,
-  div,
-  em,
-  fieldset,
-  footer,
-  form,
-  h1: heading,
-  h2: heading,
-  h3: heading,
-  h4: heading,
-  h5: heading,
-  h6: heading,
-  header,
-  hr,
-  i,
-  img,
-  input,
-  label,
-  li,
-  main,
-  nav,
-  ol,
-  optgroup,
-  option,
-  p,
-  pre,
-  section,
-  select,
-  span,
-  strong,
-  sub,
-  sup,
-  textarea,
-  ul
+  a: (a: typeof a),
```

Comment:
kind of a bummer that the annotations have to be so verbose

Reply:
I will fix this bug in `flow-api-translator`. After that, we'll be able to clean this up again.